### PR TITLE
chartboost

### PR DESCRIPTION
**Pull request by brucearctor**
## Hey, I just made a Pull Request!

Added Chartboost to list of adopters.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

Added trailing pipes, on the couple last entries, since they seemed to be missing.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -235,5 +235,6 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
-| [Syntasso](https://www.syntasso.io/)                                        | [@syntassodev](https://github.com/syntassodev)                                                                                                                                                                                                                   | Backstage is used as a optional UI for the [Kratix project](https://kratix.io), a framework for building platforms.
-| [Bankdata](https://www.bankdata.dk)                                                      | [Thomas Grønbæk](https://www.linkedin.com/in/thomas-grønbæk/)                                                                                                                                                                                                     | We are assessing Backstage as our internal developer portal @Bankdata. Goal is to reduce cognitive load and friction for developers and thereby improving developer experience, onboarding and productivity. Expect use of Software Catalog for discoverability and ownership, Tech Docs and Software Templates to integrate tooling and best practices.
```
Comment (brucearctor) [inline ADOPTERS.md:239]: trailing | was missing at EOL

File: ADOPTERS.md [hunk 2]
```diff
@@ -235,5 +235,6 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
-| [Syntasso](https://www.syntasso.io/)                                        | [@syntassodev](https://github.com/syntassodev)                                                                                                                                                                                                                   | Backstage is used as a optional UI for the [Kratix project](https://kratix.io), a framework for building platforms.
```
Comment (brucearctor) [inline ADOPTERS.md:238]: trailing pipe was missing at EOL

File: ADOPTERS.md [hunk 3]
```diff
@@ -235,5 +235,6 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
+| [Bankdata](https://www.bankdata.dk)                                                      | [Thomas Grønbæk](https://www.linkedin.com/in/thomas-grønbæk/)                                                                                                                                                                                                     | We are assessing Backstage as our internal developer portal @Bankdata. Goal is to reduce cognitive load and friction for developers and thereby improving developer experience, onboarding and productivity. Expect use of Software Catalog for discoverability and ownership, Tech Docs and Software Templates to integrate tooling and best practices. |
+| [Chartboost](https://www.chartboost.com)| [@brucearctor](https://github.com/brucearctor), [@ArtemChekunov](https://github.com/ArtemChekunov) | We are building our internal develper portal on top of Backstage.|
```
Comment (brucearctor) [inline ADOPTERS.md:240]: Adding Chartboost to the list of adopters.

File: ADOPTERS.md (hunk 1 above)
Comment (brucearctor) [reply ADOPTERS.md:239]:
```suggestion
| [Bankdata](https://www.bankdata.dk)                                                      | [Thomas Grønbæk](https://www.linkedin.com/in/thomas-grønbæk/)                                                                                                                                                                                                     | We are assessing Backstage as our internal developer portal @Bankdata. Goal is to reduce cognitive load and friction for developers and thereby improving developer experience, onboarding and productivity. Expect use of Software Catalog for discoverability and ownership, Tech Docs and Software Templates to integrate tooling and best practices.|

File: ADOPTERS.md [hunk 4]
```diff
@@ -239,3 +239,4 @@ _You can do this by using the [Adopter form](https://info.backstage.spotify.com/
...
 | [Einride](https://github.com/einride) | [@odsod](https://github.com/odsod/) | We use Backstage to create a great developer experience across Einride's entire software stack - from autonomous and electric vehicles to cloud systems - and we've developed a [Backstage Go SDK](https://github.com/einride/backstage-go) for interfacing with Backstage from our Go tooling.
```
Comment (brucearctor) [inline ADOPTERS.md:241]:
```suggestion
| [Einride](https://github.com/einride) | [@odsod](https://github.com/odsod/) | We use Backstage to create a great developer experience across Einride's entire software stack - from autonomous and electric vehicles to cloud systems - and we've developed a [Backstage Go SDK](https://github.com/einride/backstage-go) for interfacing with Backstage from our Go tooling.|

Comment (brucearctor) [reply ADOPTERS.md:241]: add a trailing pipe ( | )

**Review by jhaals — APPROVED** [review]
Thank you 👍

**Pull request merged**